recipes/websocketpp/all/conanfile.py

```diff
@@ -20,28 +19,26 @@ def _source_subfolder(self):
         return "source_subfolder"
 
     def requirements(self):
-        self.requires("openssl/1.1.1g")
+        self.requires("openssl/1.1.1h")
         self.requires("zlib/1.2.11")
         if self.options.asio == "standalone":
             self.requires("asio/1.16.1")
         else:
-            self.requires("boost/1.73.0")
+            self.requires("boost/1.74.0")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
-        self._patch_sources()
 
-    def _patch_sources(self):
-        if "patches" in self.conan_data and self.version in self.conan_data["patches"]:
-            for patch in self.conan_data["patches"][self.version]:
-                tools.patch(**patch)
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
 
     def package(self):
         self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
         # We have to copy the headers manually, since the upstream cmake.install() step doesn't do so.
-        self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + "/websocketpp")
+        self.copy(pattern=os.path.join("websocketpp","*.hpp"), dst="include", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy("*.hpp", dst="include", src=os.path.join(self._source_subfolder, "websocketpp"))
```

feels more intuitive to me, but maybe it's just an habit.

Reply:
I did not like that it needed `websocketpp` directory twice

Reply:
Ok I was disturbed by folder in the pattern, it's equivalent.